src/containers/LoginForm/index.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import * as M from "materialize-css";
```

Comment:
Change this into something more descriptive

Reply:
done